src/ui/window.c

```diff
@@ -760,16 +760,16 @@ static void win_pump_sdl(struct window *win) {
       case SDL_KEYDOWN: {
         enum keycode keycode = translate_sdl_key(ev.key.keysym);
 
-        if (keycode != K_UNKNOWN) {
```

Comment:
Looks good! Could we update the other places that were using a 0 / 1 value for keydown such that they can be mapped to triggers as well?

A local `KEY_DOWN` / `KEY_UP` or `KEY_MIN` / `KEY_MAX` define may be nice as well.